src/commands/wait_for_workflow.yml

```diff
@@ -107,6 +114,9 @@ steps:
             if [[ $running_wf_start_time < $current_workflow_start_time ]] ; then
               can_i_run="false"
               break
+            elif [[ $running_wf_start_time > $current_workflow_start_time ]] && [[ "<< parameters.auto-cancel-redundant >>" == "true" ]]; then
```

Comment:
i might be slightly mistaken, been a while, but we only filter for "running" workflows here right. Dont we need to check if there is any workflow that started after this one? not just running ones? or am I just confusing myself :)

Also if there are workflows say "a" and "b"  since we do a regex, setting the param to true could kill "b" workflows even if we are in "a", we should be careful to document this carefully so users don't get confused.